sample-job-handlers/download-file.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+set -e
+
+echo "Running download-file.sh"
+if command -v "wget" > /dev/null
```

Comment:
Are there any other mechanisms that customers use? CURL?

Reply:
Do we want to make this optional (i.e., use either curl or wget?)

Reply:
There is an issue with using CURL with S3 pre-signed URL's. If the user does not have the access to download the S3 file, instead of throwing an error and aborting operation with non-zero exit code, CURL will return exit code `0` (success) resulting in DC updating the Jobs status to `SUCCESS`. For some reason, CURL will write the exception in the output-document.  Because of this issue, user will not be notified that the download-file.sh operation failed. 
On the other hand, if wget sees access denied exception, it will abort the program with `6` and the Job status will be updated to `FAIL`.

Reply:
Are there no other indications we can use to identify whether the download using CURL was successful? Like perhaps compare the filesize of the URL loc to the output file?

Reply:
To validate file data, we need to parse the file and check the contents. But that would get complicated because the file type can be different.
To validate based on size, we need to ask user to give us original file size data. So that option is also not the best solution.